code/modules/cm_aliens/structures/special/pylon_core.dm

```diff
@@ -137,20 +140,26 @@
 /obj/effect/alien/resin/special/pylon/core/Destroy()
 	if(linked_hive)
 		linked_hive.hive_location = null
+		linked_hive.hive_structures_limit[XENO_STRUCTURE_CORE] = 0
+		xeno_message(SPAN_XENOANNOUNCE("A sudden tremor ripples through the hive... the Hive Core has been destroyed! Vengeance!"), 3, linked_hive.hivenumber)
+		addtimer(CALLBACK(src, .proc/cooldown_done, linked_hive), HIVECORE_COOLDOWN)
 		if(hardcore)
-			xeno_message(SPAN_XENOANNOUNCE("A sudden tremor ripples through the hive... the Hive Core has been destroyed! Vengeance!"), 3, linked_hive.hivenumber)
 			xeno_message(SPAN_XENOANNOUNCE("You can no longer gain new sisters or another Queen. Additionally, you are unable to heal if your Queen is dead"), 2, linked_hive.hivenumber)
 			linked_hive.hardcore = TRUE
 			linked_hive.allow_queen_evolve = FALSE
-			linked_hive.hive_structures_limit[XENO_STRUCTURE_CORE] = 0
 			linked_hive.hive_structures_limit[XENO_STRUCTURE_POOL] = 0
-
 			xeno_announcement("\The [linked_hive.name] has lost their hive core!", "everything", HIGHER_FORCE_ANNOUNCE)
 
 			if(linked_hive.spawn_pool)
 				qdel(linked_hive.spawn_pool)
-
 	. = ..()
 
+
+
+/obj/effect/alien/resin/special/pylon/core/proc/cooldown_done(var/datum/hive_status/linked_hive)
+	linked_hive.hive_structures_limit[XENO_STRUCTURE_CORE] = 1
```

Comment:
this is a very hacky way to do it and I do not like it

Reply:
Fixed, now uses its own variable instead of setting the max hive cores to 0